src/js/constants/LabelMapping.js

```diff
@@ -0,0 +1,13 @@
+var AppStatus = require("../constants/AppStatus");
+
+const LabelMapping = {
+  statusNameMapping: {
+    [AppStatus.RUNNING]: "Running",
+    [AppStatus.DEPLOYING]: "Deploying",
+    [AppStatus.SUSPENDED]: "Suspended",
+    [AppStatus.DELAYED]: "Delayed",
+    [AppStatus.WAITING]: "Waiting"
+  }
+};
+
+module.exports = Object.freeze(LabelMapping);
```

Comment:
Shouldn't this be a deepFreeze? The properties of `statusNameMapping` aren't protected otherwise.